docs/input/docs/more-info/variables.md

```diff
@@ -39,7 +39,8 @@ Running `GitVersion.exe` in your repo will show you what is available. For the
   "VersionSourceSha":"950d2f830f5a2af12a6779a48d20dcbb02351f25",
   "CommitsSinceVersionSource":1,
   "CommitsSinceVersionSourcePadded":"0001",
-  "CommitDate":"2014-03-06"
+  "CommitDate":"2014-03-06",
+  "RepositoryDirtyFlag": ""
```

Comment:
I think the `bool` version is most useful (at least that would be a use case for me). Which case would you make for counting the actual number of changes? I mean beyond just "can do" ? I don't see much value in seeing that the build was done with "N" uncommitted changes. Just knowing that it was "dirty" seems fine for me.

Reply:
Perhaps a dumb idea: should I maybe implement it both? Like one `HasUncomittedChanges` boolean and `NumberUncomittedChanges` integer?

Reply:
I like `UncommitedChanges` and think it should be a number, to accompany the rest of the variables such as `CommitsSinceVersionSource` and such so it can be used directly in version numbers easily.

Reply:
@asbjornu gotcha! Just one small related question: how should I deal with the parsing of that in `SemanticVersionBuildMetaData.Parse()` (file: `VersionCalculation/SemanticVersioning/SemanticVersionBuildMetaData.cs`)?

I previously just looked at the `OtherMetatada` property for if it was `== "Dirty"` but that won't work as simply anymore.
I suggest I just leave out that part to have something mergeable (as we're beyond the standard SemVer stuff anyway). Or do you have any concrete idea how this should be properly serialized?